frontend/src/components/homepage/sections/HomepageExperimental.tsx

```diff
@@ -0,0 +1,49 @@
+import { UswdsIconNames } from "src/types/generalTypes";
+
+import { useMessages, useTranslations } from "next-intl";
+import { Button, Grid } from "@trussworks/react-uswds";
+
+import HomePageSection from "src/components/homepage/homePageSection";
+import IconInfo from "src/components/homepage/IconInfoSection";
+
+const trySearchLink = "https://simpler.grants.gov/search";
```

Comment:
this should be simplified to `/search` so that it reflects the right environment in dev and staging. We may also want to use a <Link> tag, but it may not do much considering we already have a link to search on the page in the header

Reply:
Yes!! Apologies, I just copied the link from the figma. Updated!